librepo/downloader.c

```diff
@@ -293,7 +293,15 @@ is_max_mirrors_unlimited(const LrDownload *download)
 }
 
 static gboolean
-can_try_more(const LrDownload *download, int num_of_tried_mirrors, gboolean complete_path_or_baseurl)
+/**
+ * @brief Can be tried another mirror or retried baseurl or fullpath
+ *
+ * @param download p_download:...
```

Comment:
Is this some special syntax with the `p_download:...`? What does it do?

Reply:
not sure, this was provided directly by kdevelop

Reply:
Oh. Then it's better to remove it? The ... probably means you should add your own description? and I'm not sure what the `p_*` but I don't think we're using it, so let's not add it, it's redundant and useless?

FWIW I find the argument names self-describing, reiterating what the name already says in the argument description has little value. Since this is not a public function, I'd say a simple brief description is sufficient?